test/SlowTests/Issues/RavenDB-9727.cs

```diff
@@ -30,7 +30,7 @@ public void Can_Load_with_Argument_that_has_String_Interpolation()
                 {
                     var query = from u in session.Query<User>()
                                 where u.LastName == "Garcia"
-                                let detail = session.Load<Detail>($"details/{u.DetailShortId}")
+                                let detail = session.Load<Detail>("details/"+u.DetailShortId)
```

Comment:
@aviv86 Do we need this test, if we no longer test string interpolation anymore?